multiboot/bootstrap.md

```diff
@@ -37,11 +41,26 @@ OCB: Have existing option 1, valid 1
 OCB: Boot order has first option as the default option
 ```
 
-### Removing Bootstrap entry from BIOS
+### Removing LauncherOption entry from BIOS
 
-Because the Bootstrap entry is a protected entry when resetting NVRAM, you'll need the following settings in your config.plist:
+Because the LauncherOption entry is a protected entry when resetting NVRAM, you'll need the following settings in your config.plist:
 
-* Misc -> Security -> AllowNvramReset -> true
-* Misc -> Security -> BootProtect -> None
+* `Misc -> Security -> AllowNvramReset -> True`
+* `Misc -> Boot -> LauncherOption -> Disabled`
 
 Once these are set, you can reboot into the OpenCore picker and select the `Reset NVRAM` entry.
+
+
+## Updating BootStrap in 0.6.6
+
+For those updating to 0.6.6, you may have noticed BootStrap.efi has been removed from OpenCorePkg. This is due to changes with how OpenCore works, specifically becoming an Application instead of a driver. This means OpenCore.efi can be loaded directly and the reliance on BootStrap.efi is no longer needed.
```

Comment:
```suggestion
For those updating to 0.6.6, you may have noticed Bootstrap.efi has been removed from OpenCore. This is due to changes with how OpenCore works; specifically OpenCore is now a UEFI application instead of a driver. This means OpenCore.efi can be loaded directly and the reliance on Bootstrap.efi is no longer needed.
```